src/Backend/Modules/Location/Js/Location.js

```diff
@@ -11,7 +11,7 @@ jsBackend.location =
     init: function()
     {
         // only show a map when there are options and markers given
-        if(typeof markers != 'undefined' && typeof mapOptions != 'undefined')
+        if($('#map').length > 0 && typeof markers != 'undefined' && typeof mapOptions != 'undefined')
```

Comment:
Maybe, but that is wat is used to create the map => https://github.com/jonasdekeukelaere/forkcms/blob/fe871255770df0a93cb4cdd54f2e8d264c08e46c/src/Backend/Modules/Location/Js/Location.js#L272